velox/common/caching/SsdFile.cpp

```diff
@@ -590,8 +590,11 @@ bool SsdFile::removeFileEntries(
     for (int32_t region : toFree) {
       if (existingWritableRegions.count(region) == 0) {
         writableRegions_.push_back(region);
+        erasedRegionSizes_[region] = 0;
+        regionSizes_[region] = 0;
       }
     }
+    logEviction(toFree);
   }
 
   stats_.entriesAgedOut += entriesAgedOut;
```

Comment:
NYC: shall we add stats to record the number of writes got dropped because of no cache space? Thanks!